### DISCUSSION TITLE
Has anyone tried using this with Gatsby?

### DISCUSSION THREAD

**Discussion by cosmoburn** [wevm/wagmi#142] [Q&A]
I get it to work locally but when I deploy the build fails with this:

![image](https://user-images.githubusercontent.com/9123901/152707404-c80fabfd-f47a-4639-8599-ff13cc8ca425.png)

**Comment by tmm** [comment]
Haven't tried. Did you wrap your app (or root tree) in the [Provider](https://wagmi-xyz.vercel.app/docs/provider)?

Happy to take a look if you put together a basic CodeSandbox.
